.github/workflows/linter.yml

```diff
@@ -0,0 +1,20 @@
+name: ES Lint
+
+on:
+  push:
+    branches: [linter]
+  pull_request:
+    branches: [main]

```

Comment:
```suggestion
on:
  pull_request:
    branches: [main]
```

![Uploading Screenshot 2023-03-02 at 9.40.58 PM.png…]()
I know you added the push for testing. Looks like you don't need it anymore (now that you have a PR to main)